virt/release_notes/virt-4-17-release-notes.adoc

```diff
@@ -65,6 +65,9 @@ This release adds new features and enhancements related to the following compone
 
 * Configuring xref:../../virt/nodes/virt-node-maintenance.adoc#eviction-strategies[VM eviction strategies] for an xref:../../virt/nodes/virt-node-maintenance.adoc#virt-configuring-cluster-eviction-strategy-cli_virt-node-maintenance[entire cluster] is now generally available.
 
+//CNV-39964 GA: inferFromVolume
+* The `inferFromVolume` attribute is now supported for use with container disks. When requested, {VirtProductName} can copy the labels `instancetype.kubevirt.io/default-instancetype` and `instancetype.kubevirt.io/default-preference` from a source container disk to the boot volume of a new VM.
```

Comment:
```suggestion
* The `inferFromVolume` attribute is now supported for use with imported container disks. When requested, {VirtProductName} can copy the labels `instancetype.kubevirt.io/default-instancetype` and `instancetype.kubevirt.io/default-preference` from a source container disk to the boot volume of a new VM.
```

Reply:
Very slight but important change. Imported containerdisks can make use of the labels, but containerdisks used as ephemeral volumes can not yet.

Reply:
@0xFelix I made the change :) thanks for pointing that out!